examples/example_rock_paper_scissors.py

```diff
@@ -60,7 +60,7 @@ def run_rock_paper_scissors(population_size=100, n_iterations=200, random_seed=4
         Evolution()
         .survive(fraction=survive_fraction)
         .breed(parent_picker=pick_random, combiner=lambda x, y: x.combine(y), n_parents=2)
-        .mutate(lambda x: x.mutate())
+        .mutate(lambda x: x.mutate_with())
```

Comment:
This is the method of the `RockPaperScissorsPlayer`. If we decide to rename `mutate` to `mutate_with`, this must still be `mutate`.